### PR TITLE
Fix MaterialButton icons and shrinking with ellipsis issues on iOS

### PR DESCRIPTION
**Pull request by softlion**
What kind of change does this PR introduce? (Bug fix, feature, docs update...)

Bug Fix

### :arrow_heading_down: What is the current behavior?

- Right icons are not displayed at all.
- Left icons may not be displayed correctly.
- Sometime the text disappears and is replaced by "..."

### :new: What is the new behavior (if this is a feature change)?

All works as expected
Also the margin in ContentLayout is correctly used. 
Also the Padding has a real effect.

### :boom: Does this PR introduce a breaking change?

No

### :bug: Recommendations for testing

Run ios project and open the MaterialButton test page

### :thinking: Checklist before submitting

- [X] All projects build
- [X] Follows style guide lines 
- [ ] Relevant documentation was updated
- [X] Rebased onto current develop

### CODE REVIEW

**Comment by softlion** [comment]
Fixes https://github.com/Baseflow/XF-Material-Library/issues/368

File: Samples/MaterialMvvmSample.iOS/MaterialMvvmSample.iOS.csproj [hunk 1]
```diff
@@ -70,8 +73,9 @@
     <WarningLevel>4</WarningLevel>
     <MtouchArch>ARM64</MtouchArch>
     <ConsolePause>false</ConsolePause>
-    <CodesignKey>iPhone Developer</CodesignKey>
+    <CodesignKey>Apple Development: Benjamin Mayrargue (4FX9X5DS74)</CodesignKey>
```
Comment (martijn00) [inline Samples/MaterialMvvmSample.iOS/MaterialMvvmSample.iOS.csproj:76]: Can you revert this? And other personal things?

File: XF.Material/Platforms/Android/Renderers/MaterialButtonRenderer.cs [hunk 2]
```diff
@@ -129,10 +134,31 @@ private void SetTextColors()
             Control.SetTextColor(new ColorStateList(states, colors));
         }
 
+        /// <summary>
+        /// TODO: obsolete. Replaced by Button.CharacterSpacing.
```
Comment (martijn00) [inline XF.Material/Platforms/Android/Renderers/MaterialButtonRenderer.cs:138]: Cleanup?

**Review by martijn00 — CHANGES_REQUESTED** [review]

**Comment by softlion** [comment]
I abandon using the material buttons. They are complelty buggy and they won't fix. I just removed them all and wont fix them. You can delete the PR, it fixes some things but not all.

**Pull request closed without merging**